src/utils.js

```diff
@@ -0,0 +1,124 @@
+const fs = require("fs");
+const read = require("fs-readdir-recursive");
+
+function bundleForms() {
+  const formConfigPath = "./formSkeleton/";
+
+  let bundledFormOutput = fs.readFileSync(
+    formConfigPath + "inputFields/input-fields.ttl",
+    "utf8"
+  );
+
+  read(formConfigPath + "forms/").forEach((fileDir, i) => {
+    bundledFormOutput += fs.readFileSync(
+      formConfigPath + "forms/" + fileDir,
+      "utf8"
+    );
+  });
+
+  return bundledFormOutput;
+}
+
+function clearDistFolder() {
+  if (fs.existsSync("./dist")) {
+    fs.rmSync("./dist", { recursive: true });
+  }
+}
+
+function createFormTtlFile({ project, fileName, bundledFormOutput }) {
+  fs.mkdirSync(`./dist/${project.name}/config/semanticForms`, {
```

Comment:
If I look at Loket, these `.ttl` files are found under `/config/semantic-forms`. So it would be nice if it was the same here. If it is different per project. Maybe consider a similar behavior as the graph per project?

Reply:
Ah, I didn't verify that. I followed what [was written in the readme](https://github.com/lblod/manage-submission-form-tooling/pull/10/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L90). I'll add it to the project config :+1:.